Sources_v3/StreamChat/ChatClient_Tests.swift

```diff
@@ -334,28 +344,26 @@ class ChatClient_Tests: StressTestCase {
         // Prepare a test worker
         class TestWorker: Worker {
             var init_database: DatabaseContainer?
-            var init_webSocketClient: WebSocketClient?
             var init_apiClient: APIClient?
             
-            override init(database: DatabaseContainer, webSocketClient: WebSocketClient, apiClient: APIClient) {
+            override init(database: DatabaseContainer, apiClient: APIClient) {
                 init_database = database
-                init_webSocketClient = webSocketClient
                 init_apiClient = apiClient
                 
-                super.init(database: database, webSocketClient: webSocketClient, apiClient: apiClient)
+                super.init(database: database, apiClient: apiClient)
             }
         }
         
         // Create a Client instance and check the TestWorker is initialized properly
         let client = _ChatClient(
             config: config,
             workerBuilders: [TestWorker.init],
+            eventWorkerBuilders: [],
             environment: testEnv.environment
         )
         
         let testWorker = client.backgroundWorkers.first as? TestWorker
         XCTAssert(testWorker?.init_database is DatabaseContainerMock)
-        XCTAssert(testWorker?.init_webSocketClient is WebSocketClientMock)
```

Comment:
This should be probably replaced with a test for proper initialization of event workers?